vickev/howdypix
# The user can see a specific image

## Description

The user can see a specific image in big in a separate page. There is only the picture displayed, nothing else for now.





## How to test?

- Check-out this branch.
- Run the server and browse an image
- The image should be displayed

## Checklist
